vms/platformvm/state/state.go

```diff
@@ -401,10 +392,10 @@ func new(
 	bootstrapped *utils.Atomic[bool],
 	trackChecksums bool,
 ) (*state, error) {
-	blockCache, err := metercacher.New(
+	blockCache, err := metercacher.New[ids.ID, blocks.Block](
 		"block_cache",
 		metricsReg,
-		cache.NewSizedLRU[ids.ID, *stateBlk](blockCacheSize, stateBlkSize),
+		cache.NewSizedLRU[ids.ID, blocks.Block](blockCacheSize, blocks.Size),
```

Comment:
Any particular rationale between this switch (*stateBlk to blocks.Block)? Just that we don't need to store status anymore?

Reply:
Yeah everything we store here is accepted

Reply:
Yeah - we don't use the status anymore. When we prune all the statuses from disk - we'll completely remove the `stateBlk` struct.